Offroad-iOS/Offroad-iOS/Presentation/Character/ChoosingCharacter/ViewController/ChoosingCharacterViewController.swift

```diff
@@ -76,23 +78,35 @@ final class ChoosingCharacterViewController: UIViewController {
             switch response {
             case .success(let data):
                 let count = data?.data.characters.count ?? 0
+                let firstCharacterImageURL = data?.data.characters[0].characterBaseImageUrl ?? ""
+                let lastCharacterImageURL = data?.data.characters[count - 1].characterBaseImageUrl ?? ""
                 
                 self.characterInfoModelList = data?.data.characters
                 
-                self.extendedCharacterImageList.insert(data?.data.characters[count - 1].characterBaseImageUrl ?? "", at: 0)
+                self.extendedCharacterImageList.insert(self.convertSvgURLToUIImage(svgUrlString: lastCharacterImageURL), at: 0)
                 for character in data?.data.characters ?? [CharacterList]() {
-                    self.extendedCharacterImageList.append(character.characterBaseImageUrl)
+                    let characterImageURL = character.characterBaseImageUrl
+                    
+                    self.extendedCharacterImageList.append(self.convertSvgURLToUIImage(svgUrlString: characterImageURL))
                     self.characterNames.append(character.name)
                     self.characterDiscriptions.append(character.description)
                 }
-                self.extendedCharacterImageList.append(data?.data.characters[0].characterBaseImageUrl ?? "")
+                self.extendedCharacterImageList.append(self.convertSvgURLToUIImage(svgUrlString: firstCharacterImageURL))
                 
             default:
                 break
             }
         }
     }
     
+    private func convertSvgURLToUIImage(svgUrlString: String) -> UIImage {
+        guard let svgURL = URL(string: svgUrlString) else { return UIImage() }
+        
+        guard let svgImage = SVGKImage(contentsOf: svgURL) else { return UIImage() }
```

Comment:
P2
해당 생성자는 네트워크 통신을 통해 이미지를 받아오는 역할을 하여 비동기적으로 동작하는 것으로 보입니다. 
(실제로 콘솔에서 빨간색으로 경고가 뜨더라구요. 아마 메인쓰레드에서 이미지를 받아와서 그런 것 같습니다.
경고의 내용은 다음과 같습니다. 
"Synchronous URL loading of { image URL } should not occur on this application's main thread as it may lead to UI unresponsiveness. Please switch to an asynchronous networking API such as URLSession.")
이미지의 데이터 크기가 크지 않아서 지금은 큰 문제가 되지 않을 것으로 예상되지만, 혹시라도 이미지의 크기가 크거나 인터넷 환경이 좋지 않은 경우 문제가 생길 수도 있기 때문에, 추후 비동기 로직을 적용하여 코드를 구현하면 좋을 것 같다는 말씀 드립니다~

Reply:
이 부분 추후 반영하겠습니다!